1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -1,3 +1,21 @@
+const text = document.getElementById("quote");
+const author = document.getElementById("author");
+
+function newQuote(){
+  const randomobj = Math.floor(Math.random() * quotes.length);
```

Comment:
it would be better to use `let` instead of `const` because the value of randomobj changes on every function call.